src/Adapter/Module/ModulePresenter.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
```

Comment:
Will be done, but it appears that another classes do not have their header too (this file comes from a copy-paste)